Mods.md

```diff
@@ -0,0 +1,47 @@
+This guide will describe how mods and their triggers attach to these projects.
+
+### Table of Contents
+1. [ModAttachr](#modattachr)
```

Comment:
Can you split part of the content into a second section, something like "Event Hooks"? Partially to justify having a ToC, partly to split the logical sections.